### DISCUSSION TITLE
Can we have one control plane for multiple k8s clusters?

### DISCUSSION THREAD

**Discussion by gopimandala** [chaos-mesh/chaos-mesh#3419] [Q&A] [answered]
Hi, If we have multiple k8s clusters, do we need to install full chaos mesh on each of them? or can we have one chaos mesh control plane for multiple clusters? I am asking bcos i couldn't find any agent installation and registration in the documentation. Cheers.

**Comment by STRRL** [comment, marked as answer]
We are working on the multi-cluster solution for Chaos Mesh! I think it would still take a while.

References:

- https://github.com/chaos-mesh/chaos-mesh/issues/3341
